agent/exec/container/executor.go

```diff
@@ -41,12 +41,35 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 		}
 	}
 
+	// add v1 plugins to 'plugins'
 	addPlugins("Volume", info.Plugins.Volume)
```

Comment:
It feels weird not to add them imho. Why would only built-in drivers + v1 plugins appear there?